src/Sylius/Bundle/CoreBundle/Processor/DummyCatalogPromotionProcessor.php

```diff
@@ -16,9 +16,9 @@
 use Doctrine\ORM\EntityManagerInterface;
 use Sylius\Bundle\CoreBundle\Applicator\CatalogPromotionApplicatorInterface;
 use Sylius\Component\Core\Model\CatalogPromotionInterface;
-use Sylius\Component\Core\Model\ProductInterface;
 use Sylius\Component\Core\Model\ProductVariantInterface;
 use Sylius\Component\Core\Provider\CatalogPromotionVariantsProviderInterface;
+use Sylius\Component\Promotion\Model\CatalogPromotionActionInterface;
 
 final class DummyCatalogPromotionProcessor implements CatalogPromotionProcessorInterface
```

Comment:
we have merged rules, and now we are merging action, is it still Dummy processor?

Reply:
It's not dummy anymore! 🎉 🎉 🎉